### DISCUSSION TITLE
sometimes: grpc_status:14, grpc_message:"recvmsg:Operation timed out"

### DISCUSSION THREAD

**Discussion by jllopd** [qdrant/qdrant#3129] [Q&A]
Hi,

I am using qdrant docker image and doing searches from langchain using grpc.

When the server is not receiving any request for some time, the first requests got this error:

Qdrant similarity_search error: <AioRpcError of RPC that terminated with:
        status = StatusCode.UNAVAILABLE
        details = "recvmsg:Operation timed out"
        debug_error_string = "UNKNOWN:Error received from peer ipv4:XX.XX.XX.XX:6334 {created_time:"2023-12-02T00:33:35.130717+01:00", grpc_status:14, grpc_message:"recvmsg:Operation timed out"}"

The nexts requests works fine until the server is idle for some time again.

Any ideas?

Thanks.

**Comment by gokturkDev** [comment]
I am facing this as well with weaviate
